backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/KonanConfig.kt

```diff
@@ -106,17 +108,33 @@ class KonanConfig(val project: Project, val configuration: CompilerConfiguration
         return resolvedLibraries.filterRoots { (!it.isDefault && !this.purgeUserLibs) || it.isNeededForLink }.getFullList()
     }
 
-    internal val defaultNativeLibraries = 
-        if (produce == CompilerOutputKind.PROGRAM) 
-            File(distribution.defaultNatives(target)).listFiles.map { it.absolutePath } 
-        else emptyList()
+    internal val defaultNativeLibraries: List<String> = mutableListOf<String>().apply {
+        add(if (debug) "debug.bc" else "release.bc")
+
+        if (produce == CompilerOutputKind.PROGRAM) {
+            add("start.bc")
+            add("launcher.bc")
```

Comment:
We'd better keep the details of distribution file names and locations in the Distribution, I think. The way we reference runtime.bc there.